rviz_default_plugins/CMakeLists.txt

```diff
@@ -9,6 +9,13 @@ endif()
 
 option(EnableDisplayTests "EnableDisplayTests")
 set(DisplayTests "False" CACHE STRING "DisplayTestsVariable")
+option(EnableVisualTests "decides whether or not to enable the tests")
+
+add_definitions(-D_BUILD_DIR_PATH="${CMAKE_CURRENT_BINARY_DIR}")
```

Comment:
Looks like this isn't being used anywhere else in the pr?

Reply:
I see, it's fine. You can change it if you like, but as long as it's documented it's fine as-is for me.